app/templates/partials/_download_list.html

```
@@ -18,127 +18,45 @@
-          />
-        </svg>
-      </button>
-    </div>
-  </div>
+    {% include 'partials/_download_item.html' %}
   {% endfor %}
 </div>
 {% else %}
-<div id="download-rows">
+<div
+  id="download-rows"
```

HTMLHint (1.9.2)

[error] The id value [ download-rows ] must be unique.

(id-unique)